runtime/src/lib.rs

```diff
@@ -355,16 +350,28 @@ impl pallet_timestamp::Config for Runtime {
 	type Moment = u64;
 	type OnTimestampSet = ();
 	type MinimumPeriod = MinimumPeriod;
-	type WeightInfo = ();
+	type WeightInfo = (); // TODO: Should this have a weight?
```

Comment:
Yes, we need the custom weights. In order to get them we need to run the benchmarks.

Reply:
What's the process for this and who can we ping about it?